R/utilities.r

```diff
@@ -107,6 +100,10 @@ remove_missing <- function(df, na.rm = FALSE, vars = names(df), name = "",
 
   df
 }
+detect_missing <- function(df, vars, finite = FALSE) {
+  vars <- intersect(vars, names(df))
+  !cases(df[, vars, drop = FALSE], if (finite) is_finite else is_complete)
```

Comment:
Is this line correct? Where does `is_complete` come from?

Reply:
It's a function defined in `utilitites.r`. `detect_missing()` is more or less lifted from `remove_missing()` so that `sf_grobs()` can use the same logic as `handle_na()`